CMakeLists.txt

```diff
@@ -29,15 +29,6 @@ list(APPEND LIBS ${G3FILE_ROTATE})
 find_library(FILEIO FileIO PATHS /usr/local/probe/lib)
 list(APPEND LIBS ${FILEIO})
 
-find_library(BOOST_THREAD boost_thread PATHS /usr/local/probe/lib )
-list(APPEND LIBS ${BOOST_THREAD})
-
-find_library(BOOST_SYSTEM boost_thread PATHS /usr/local/probe/lib )
-list(APPEND LIBS ${BOOST_SYSTEM})
-
-find_library(TCMALLOC tcmalloc PATHS /usr/local/probe/lib )
```

Comment:
@john-gress :  I don't think it is needed for tcmalloc  to be linked for this library. If you disagree then I could make it so that the tcmalloc library is passed in through a cmake argument.